packages/app/src/systems/Account/components/BalanceWidget/BalanceWidget.tsx

```diff
@@ -55,13 +60,15 @@ export function BalanceWidget({
   onChangeVisibility,
 }: BalanceWidgetProps) {
   const { handlers } = useAccounts();
-
+  const totalBalanceInUsd = useAccountTotalBalance() ?? 0;
   const { original, tooltip } = useMemo(() => {
     return formatBalance(account?.balance, decimals);
   }, [account]);
 
   if (isLoading || !account) return <BalanceWidget.Loader />;
 
+  const totalValue = `$${totalBalanceInUsd?.toFixed?.(2) || '0.00'}`;
```

Comment:
will this have the same format behavior as before? by the looks of it it's not gonna have thousand separators.